src/schema/SchemaGenerator.ts

```diff
@@ -30,20 +30,26 @@ export class SchemaGenerator {
       // This file has been auto-generated by Warthog.  Do not update directly as it
       // will be re-written.  If you need to change this file, update models or add
       // new TypeGraphQL objects
+      // prettier-ignore
       // @ts-ignore
       import { DateResolver as Date } from 'graphql-scalars';
+      // prettier-ignore
       // @ts-ignore
       import { GraphQLID as ID } from 'graphql';
+      // prettier-ignore
       // @ts-ignore
       import { ArgsType, Field as TypeGraphQLField, Float, InputType as TypeGraphQLInputType, Int } from 'type-graphql';
+      // prettier-ignore
       // @ts-ignore
       import { registerEnumType, GraphQLISODateTime as DateTime } from "type-graphql";
 
+      // prettier-ignore
       // @ts-ignore eslint-disable-next-line @typescript-eslint/no-var-requires
       const { GraphQLJSONObject } = require('graphql-type-json');
-
+      // prettier-ignore
       // @ts-ignore
-      import { BaseWhereInput, JsonObject, PaginationArgs, DateOnlyString, DateTimeString } from '${warthogImportPath}';      
+      import { BaseWhereInput, JsonObject, PaginationArgs, DateOnlyString, DateTimeString } from '${warthogImportPath}';
+
```

Comment:
This makes life easier by keeping `prettier` from rewriting these imports as multi-line imports and thus not applying the `// @ts-ignore` to any of the actual imports (and as a result breaking compilation when one of the imports is unused).

Reply:
Unfortunately `prettier` doesn't currently support anything like `prettier-ignore-start`/`prettier-ignore-stop` outside of markdown.